### DISCUSSION TITLE
How can I reused the weight from a trained model, then apply that weight to a new model with similar data with slight difference in IC/BC?

### DISCUSSION THREAD

**Discussion by Nhatkhang** [lululxvi/deepxde#1709] [Q&A] [answered]
Hi Dev Team, 
I am trying to do model transfer by assigning the weight of pre-trained model to a new model with same NN structure with a slight difference in data as the IC/BC changes a bit.

**Comment by lululxvi** [comment, marked as answer]
Please read FAQ. Keywords: "save" and "load".
